pkg/compliance/resource.go

```diff
@@ -129,8 +150,8 @@ const (
 	// PropertyKindAttribute describes an attribute
 	PropertyKindAttribute = "attribute"
 
-	// PropertyKindJSONPath describes a JSONPath query
-	PropertyKindJSONPath = "jsonpath"
+	// PropertyKindJSONQuery describes a JSON query (jq syntax)
+	PropertyKindJSONQuery = "jsonquery"
```

Comment:
I wonder if we could still call it `jsonpath` even if using gojq, is the syntax dramatically different? The reason I ask is that in Kubernetes ecosystem jsonpath seems to be prevalent - e.g. in kubectl output, CRDs - additional printer columns, etc.

Reply:
It's true, it was just to emphasise on the extra capabilities added by jquery over "simple" jsonpath, but I can change back it's you think `jsonpath` is clearer

Reply:
We could alias it btw.